src/bioutils/sequences.py

```diff
@@ -229,6 +229,9 @@
 dna_to_aa1_vmito["ATA"] = "M"
 dna_to_aa1_vmito["TGA"] = "W"
 
+# Alternative start codons that are only applied if the starts_at_first_codon flag is set to True
+alt_init_codons_vmito = {"ATT": "M"}
```

Comment:
This is the alternate start codon for human. I don't think we need full support for all species, but perhaps a comment is needed so users know this applies to human mitochondrial translation. Not just vertebrate.